static/js/components/search/LearnerResult.js

```diff
@@ -41,17 +41,20 @@ class LearnerResult extends SearchkitComponent {
           <ProfileImage profile={profile} useSmall={true} />
         </Cell>
         <Cell
-          col={5}
-          className="learner-name centered"
+          col={4}
+          className="learner-name"
           onMouseLeave={() => setLearnerChipVisibility(null)}
           onMouseEnter={() => setLearnerChipVisibility(profile.username)}
         >
           <span className="display-name">
             { highlight(getUserDisplayName(profile), this.searchkit.state.q) }
           </span>
+          <span className="user-name">
+            { profile.username }
```

Comment:
Can we highlight this value like we do for the other fields? See `display-name` above for an example